gen-ai-rag-vertex-ai-vector-search/serving.tf

```diff
@@ -0,0 +1,51 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Front end interface serving
+
+resource "google_cloud_run_v2_service" "serving" {
+  name     = "serving-${local.unique_str}"
+  location = var.region
+
```

Comment:
To let users delete the service by using `terraform destroy`, we need to set `deletion_protection = false`.